README.md

```diff
@@ -163,150 +162,97 @@ déployer les services :
 En outre, il vous permet d’instancier un environnement de développement local pour ces services.
 Pour ce faire merci de prendre connaissance de la suite du document (en anglais).
 
-## Install
+## Installation
 
-### Dependencies setup
+### Datapass
 
-- [VirtualBox \^5.2.10](https://www.virtualbox.org)
-- [Vagrant \^2.1.1](https://www.vagrantup.com)
-- NFS
+#### Dependencies setup
+
+- [Docker](https://www.docker.com/)
+- [Docker compose](https://docs.docker.com/compose/install/)
 - [Ansible 2.9.10](https://www.ansible.com/)
+- [NodeJS >=16](https://nodejs.org/)
 
-### DataPass local provisioning
+#### Local environment
 
 Clone the repo:
 
 ```bash
 git clone git@github.com:betagouv/datapass.git
 ```
 
-Add the following hosts in `/etc/hosts`:
+Ask a colleague to give you the backend secrets stored in the `signup-back/.env.local` file.
```

Comment:
what secret are you talking about ? is it for api-auth client ?

Reply:
yes, the `DATAPASS_OAUTH_CLIENT_ID` and `DATAPASS_OAUTH_CLIENT_SECRET` variables